lua/ulx/modules/sh/cfc_constraint_checker.lua

```diff
@@ -94,7 +94,7 @@ local function countConstraints( plys )
     return perPlyConstraints
 end
 
-local function compileConstraintResults( _, ply, constraintCounts )
+local function getPlyCountsMsgData( ply, constraintCounts )
```

Comment:
I think naming it this makes it clear that the purpose of this functions is creating args for MsgC to visualize the data